README.md

```diff
@@ -36,16 +47,15 @@ New GA releases mainly happen on master branch, a `stable-*` maintenance branch
 Only pre-releases that are SOAKed for at least two weeks should be used as stable releases. After the soaking period was successful, follow these steps:
 
 1. fetch latest repo state: `git fetch` and make sure you're on `master`
-1. update kommanders `appVersion` to be stable by removing the pre-release suffix and commit that change to `master` (remember to also update revision)
-1. apply and push tags
-   1. "base" semver based tag: e.g. `git tag v1.1.0 && git push origin v1.1.0`
-   1. "consumable" stable tag(s) for supported k8s versions: e.g. `git tag stable-1.16-1.1.0 && git push origin stable-1.16-1.1.0`
```

Comment:
Do we still need these tags?

Reply:
AFAIK no. we're not using any kind of automatic detection anyway,. we need to get https://github.com/mesosphere/ksphere-platform/pull/118 in 🙈